### PR TITLE
This is an example of PRing a guide for Installing Node.js

### PR DESCRIPTION
**Pull request by ayliao**
* It only has an example commit so far 💁🏽

### CODE REVIEW

File: documentation/0000-installing-nodejs.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+- **Start Date:** (fill me in with today's date, YYYY-MM-DD)
```
Comment (keywordnew) [inline documentation/0000-installing-nodejs.md:1]: It would be helpful to have a start date!

**Review by keywordnew — CHANGES_REQUESTED** [review]
Great start, please have a look at the comments, and we'll help you get this PR in!

🎉🎉🎉

File: documentation/0000-installing-nodejs.md (hunk 1 above)
Comment (ayliao) [reply documentation/0000-installing-nodejs.md:1]: hey, youre the one who wrote thus!

**Comment by ayliao** [comment]
closing this, as this was just used for a class example!

**Pull request closed without merging**